packages/web-ui/src/ds/atoms/Text/index.tsx

```diff
@@ -275,7 +275,7 @@ namespace Text {
       children,
       color = 'foreground',
       overflow = 'auto',
-      whiteSpace = 'normal',
+      whiteSpace = 'pre',
```

Comment:
Changed this default attribute for `Text.Mono`. This makes more sense for a code snippet or whatever regular use case does `Text.Mono` have, and no other component is using it yet.